src/include/rbd/librbd.hpp

```diff
@@ -142,6 +157,14 @@ class CEPH_RBD_API RBD
 	     IoCtx& c_ioctx, const char *c_name, ImageOptions& opts);
   int remove(IoCtx& io_ctx, const char *name);
   int remove_with_progress(IoCtx& io_ctx, const char *name, ProgressContext& pctx);
+  int trash_move(IoCtx &io_ctx, const char *name, uint64_t delay);
+  int trash_list(IoCtx &io_ctx,
+                 std::map<std::string, trash_image_info_t> &entries);
+  int trash_purge(IoCtx &io_ctx, const char *image_id, bool force);
```

Comment:
I think I originally misread this as a "purge all images in the trash" API command vs a remove an individual item from the trash. Since we already use purge to denote removing multiple items (i.e. rbd snap purge), can we rename this (and the associated CLI command) to trash_remove?